src/AdminConsole/Pages/Account/Login.cshtml

```diff
@@ -82,11 +82,12 @@
             setup() {
 
                 async function signIn(token) {
-                    var res = await fetch("/passwordless/verify-signin", {
+                    var res = await fetch("/passwordless-login", {
                         method: "POST",
-                        body: new URLSearchParams({ token }),

```

Comment:
While not critical, I find using URLSearchParams to be a bit easier than JSON.

It doesn't require changing ContentType and you can also bind to non-complex objects on the endpoints, such as (string token) without the wrapper DTO.